.clang-tidy

```diff
@@ -5,7 +5,6 @@ Checks: >
     modernize-concat-nested-namespaces,
     modernize-deprecated-headers,
     modernize-deprecated-ios-base-aliases,
-    modernize-loop-convert,
```

Comment:
not supported in c++20?

Reply:
it is so strict that we should change too mangy loops.